ui/src/main/resources/edu/wpi/grip/ui/templates/java/Pipeline.vm

```diff
@@ -0,0 +1,93 @@
+#**
+This creates the complete java pipeline.
+*##set($needsMut = false)
+#set($interpolation = false)
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.HashMap;
+
+import org.opencv.core.*;
+import org.opencv.core.Core.*;
+import org.opencv.features2d.FeatureDetector;
+import org.opencv.imgcodecs.Imgcodecs;
+import org.opencv.imgproc.*;
+
+/**
+* Pipeline class.
+*
+* <P>Autogenerated pipeline from GRIP.
```

Comment:
The black-box idea has not changed. Currently the plan is that the user will select the location where they want to save the file when they generate the code. If there is another file with the same name it will give a warning that the file will be overwritten in a similar way to how saving normal files works. We will also add some comments at the top that explain how to use the generated class including any needed warnings.